admin/modules/config/settings.php

```diff
@@ -1025,6 +1025,13 @@
 			}
 		}
 
+		// Verify for admin email that can't be empty
+		if(isset($mybb->input['upsetting']['adminemail']) && !filter_var($mybb->input['upsetting']['adminemail'], FILTER_VALIDATE_EMAIL))
+		{
+			flash_message($lang->error_admin_email_settings_empty, "error");
+			admin_redirect("index.php?module=config-settings&action=change&gid=".$gid);
```

Comment:
```suggestion

			unset($mybb->input['upsetting']['adminemail']);
			$lang->success_settings_updated .= $lang->error_admin_email_settings_empty;
```

Reply:
@JordanMussi what about this? This looks inconsistent with how we handle the login methods and emails settings.

![image](https://user-images.githubusercontent.com/1786584/85985109-12870a80-b99f-11ea-9680-cf5031c1768f.png)

Reply:
Please note that the current approach ignores form changes entirely when redirecting the user to the same page.